.travis.yml

```
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
-  - '9'
+  - '10'
+  - '12'
```

Maybe worth explicitly adding 13 as well?

---

Sure

---

`node` or `latest` may be better than `13` (fail early).

---

I added both